src/test/data/JsonSerializablePropertyDirectoryTest/typicalPropertyDirectory.json

```diff
@@ -4,13 +4,15 @@
     {
       "name": "Pinnacle@Duxton",
       "address": "123, Jurong West Ave 6, #08-111",
+      "price": "1,000,000",
```

Comment:
Will the data automatically include commas when saving? I think we should leave it as a number.